docs/application/native/guides/alarm/alarms.md

```diff
@@ -21,7 +21,7 @@ The main features of the Alarm API include:
 
 - Managing existing alarms
 
-  You can [list all scheduled alarms and cancel them](#scenario_4).
+  You can [list all scheduled alarms and update or cancel them](#scenario_4).
```

Comment:
Using ‘and’ and ‘or’ together is creating confusion while reading the sentence from an end user perspective. Can we re-write the sentence as: 
You can [list all the scheduled alarms, and also update or cancel them](#scenario_4).

Kindly, check for technical accuracy before incorporating the suggested re-write for the same.

Reply:
Done